apps/console/src/features/organizations/api/organization.ts

```diff
@@ -348,3 +348,29 @@ export const getSharedOrganizations = (
             return Promise.reject(error);
         });
 };
+
+/**
+ * Gets the super organization id of the current user.
+ *
+ * @returns {Promise<string>} The super organization id of teh user.
+ */
+export const getUserSuperOrganization = (): Promise<string> => {
```

Comment:
Shall we consider SWR?
https://github.com/wso2/identity-apps/blob/master/docs/write-code/DATA_FETCHING.md#userequest